test/test_model.py

```diff
@@ -23,583 +24,6 @@
 CN70 = [0.000, 0.030, 0.060, 0.110, 0.170, 0.240, 0.460, 0.710, 1.010, 1.330, 1.670, 2.040, 2.810, 3.620, 4.460, 5.330, 6.220, 7.130, 8.050, 8.980, 9.910, 10.85]  # noqa
 CN80 = [0.080, 0.150, 0.240, 0.340, 0.440, 0.560, 0.890, 1.250, 1.640, 2.040, 2.460, 2.890, 3.780, 4.690, 5.630, 6.570, 7.520, 8.480, 9.450, 10.42, 11.39, 12.37]  # noqa
 CN90 = [0.320, 0.460, 0.610, 0.760, 0.930, 1.090, 1.530, 1.980, 2.450, 2.920, 3.400, 3.880, 4.850, 5.820, 6.810, 7.790, 8.780, 9.770, 10.76, 11.76, 12.75, 13.74]  # noqa
-# INPUT and OUTPUT are data that were emailed to Azavea in a
-# spreadsheet for testing the TR-55 model implementation.
-INPUT = [
```

Comment:
No, it didn't need to be removed, but it didn't seem like it was proving much.  Essentially, it was just showing that the output was similar to that from the spreadsheet.  In other words, the numeric bounds were essentially just arbitrary numbers.